tests/integration_tests/test_wave_01_02_03.py

```diff
@@ -2,8 +2,8 @@
 from swap_meet.vendor import Vendor
 from swap_meet.item import Item
 
-@pytest.mark.skip
-@pytest.mark.integration_test
+# @pytest.mark.skip
+# @pytest.mark.integration_test
```

Comment:
Don't comment out the `integration_test` decorator. Doing so doesn't hurt anything here, but marking a test with `integration_test` can tell automated test runners what sort of test a test is, and this information can be used to decide when or how often to to run those tests.